app/views/staff/pages/themes/default/home.html.erb

```diff
@@ -0,0 +1,95 @@
+      <p class="mb-6 leading-5">The Hyatt Regency in Denver, which will serve as our conference hotel, has a reservation block for us, with rooms starting at $189/night (plus fees) until October 28th and then $229/night. However, rates may increase if the conference block sells out, so be sure to book your room as soon as possible.</p>
+      <a href="#" class="bg-gray-200 rounded-lg p-4 text-sm font-bold">Make a Reservation</a>
+    </div>
+  </div>
+  <div><%= image_tag 'themes/default/image-placeholder.png', class: "mx-auto" %></div>
+  <div><%= image_tag 'themes/default/image-placeholder.png', class: "mx-auto" %></div>
+</div>
+
+<!-- Attraction Component -->
+<h4 class="text-center text-3xl font-bold tracking-wider mb-6">Attractions</h4>
+<div class="grid grid-cols-2 grid-rows-2 gap-x-3 gap-y-4 max-w-4xl mx-auto mb-6">
+  <div class="self-end"><%= image_tag 'themes/default/image-placeholder.png', class: "mx-auto" %></div>
+  <div class="order-last lg:order-none"><%= image_tag 'themes/default/image-placeholder.png', class: "mx-auto" %></div>
+  <div class="row-span-2 lg:row-span-1 lg:col-span-2 lg:columns-3 flex flex-col justify-center">
+    <p class="mb-3">Denver is an outdoor city with a vibrant, walkable downtown offering urban adventures for all ages. Within a one-mile radius, you can kayak on the Platte River; explore some of the city's best museums; shop at art galleries and boutiques; see a Broadway touring show; enjoy live music; ride a rollercoaster or cool off in a water park; and experience exciting professional sports. Experience Denver's top chef-driven restaurants and innovative craft breweries. The Denver Beer Trail is a great guide to sampling those local brews.</p>
+    <p class="mb-3">Downtown's free 16th Street Mall shuttle makes it easy to get around. Visit Larimer Square and the renovated Denver Union Station in historic Lower Downtown (LoDo), where Victorian buildings now house restaurants, galleries, shops, bars, brewpubs and coffee houses, or head to Denver Pavilions where you'll find shops, restaurants, bars and a movie theater.<br />Discover the downtown area, and don't miss the chance for a photo with everyone's favorite Blue Bear public art sculpture at the Colorado Convention Center.</p>
+    <p class="mb-3">Here are more of Denver's Mile High Must-Sees.</p>
+  </div>
+</div>
+
+<!-- Opportunities Component -->
+<h4 class="text-center text-3xl font-bold tracking-wider mb-6">Opportunities</h4>
+<div class="mb-6 flex flex-col lg:flex-row items-center lg:justify-center gap-4 max-w-5xl mx-auto mb-12">
+  <div class="w-full lg:w-2/5 flex flex-row lg:flex-col">
+  <%= image_tag 'themes/default/image-placeholder.png', class: "w-1/3 lg:w-full mr-3 block mb-4" %>
+    <div>
+      <h6 class="text-xl font-bold mb-3">Speak at RubyConf</h6>
```

Comment:
@cody it seems like these elements about speaking and CFPs could be made dynamic?

Reply:
👍